library/src/main/java/com/novoda/downloadmanager/DownloadsNetworkRecoveryCreator.java

```diff
@@ -2,23 +2,27 @@
 
 import android.content.Context;
 
-class DownloadsNetworkRecoveryCreator {
+final class DownloadsNetworkRecoveryCreator {
 
-    private static DownloadsNetworkRecovery INSTANCE;
+    private DownloadsNetworkRecoveryCreator() {
```

Comment:
should the constructor be after all class fields?

Reply:
4be4c57